docs/guides/log_redirection.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: log-redirection
+title: Log redirection
+description: Redirect logs from another Actor run
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+
+In some situations, one Actor is going to start one or more other Actors and wait for them to finish and produce some results. In such cases, you might want to redirect the logs of the started Actors' runs back to the parent Actor run, so that you can see the progress of the started Actors' runs in the parent Actor's logs. This guide will show possibilities on how to do it.
+
+### Redirecting logs from `Actor.call`
+
+Typical use case for log redirection is to call another Actor using the [`Actor.call()`](/reference/class/Actor#call) method. This method has an optional logger argument, which is by default set to the `default` literal. This means that the logs of the called Actor will be automatically redirected to the parent Actor's logs with default formatting and filtering. If you set the logger argument to `null`, then no log redirection happens. The third option is to pass your own `Log` instance with the possibility to define your specific logging. Below you can see those three possible ways of log redirection when starting another Actor run through Actor.call.
+
+```javascript
+import { Actor } from 'apify';
+import {LoggerActorRedirect} from 'apify-client';
```

Comment:
```suggestion
import { LoggerActorRedirect } from 'apify-client';
```

please lint / format the examples (this might not be the only formatting issue)